src/OpenLoco/CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ set(OLOCO_SOURCES
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Interop/Hooks.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/Intro.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/src/LastGameOptionManager.cpp"
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/LastMapWindowAttributes.cpp"
```

Comment:
After changing this to the Ui namespace you might as well move this to the Ui folder.